src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs

```diff
@@ -305,8 +310,9 @@ public ImmutableArray<TaggedText> Format(string rawXmlText, ISymbol symbol, Sema
 
         private static void AppendTextFromNode(FormatterState state, XNode node, Compilation compilation)
         {
-            if (node.NodeType == XmlNodeType.Text)
+            if (node.NodeType is XmlNodeType.Text or XmlNodeType.CDATA)
             {
+                // cast is safe since XCData inherits XText
```

Comment:
should the above just be `if (node is XText xtext)` ?

Reply:
➡️ Both approaches are fine. There isn't really a downside to the current approach so I prefer to not make a change here unless it's being done along with other changes.